https://github.com/Grumbel/tinycmmc/pull/2
https://github.com/Grumbel/tinycmmc/commit/8238a6c1b90536e211fddf356dc3af26ea7c2f2c

From 01e81cdeb672d57f050594517a3ec4de4298615c Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Wed, 26 Mar 2025 07:45:32 -0700
Subject: [PATCH] cmake: require 3.10 as the minimum version

With cmake 4 it now errors unless the minimum version is at least 3.5
and it will warn if the version is not at least 3.10.

Gentoo-Issue: https://bugs.gentoo.org/952072
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2344961..12af9d2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@
 #    misrepresented as being the original software.
 # 3. This notice may not be removed or altered from any source distribution.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.10)
 project(tinycmmc VERSION 0.1.0)
 
 include(GNUInstallDirs)
